### DISCUSSION TITLE
Recommendations/suggestions for imroving the sessions

### DISCUSSION THREAD

**Discussion by akash-coded** [akash-coded/spring-framework#111] [Ideas]
Please comment your ideas or your experience or feedback on the session conducted those far.

**Comment by Shraddha-salokhe** [comment]
you spoke too fast...and so I didnt get clearly all concepts. I am totally begginer in this field sir.

**Comment by Bhaba123** [comment]
Sir, u need to go a little slow with the session ,we are getting confused when u go so fast

**Reply by Shraddha-salokhe** [reply]
right

**Comment by rajatccsu** [comment]
Sir, u need to go a little slow with the session ,we are getting confused when u go so fast

**Comment by rockyrkl99** [comment]
package rocky;

public class Main {
    public static void main(String[] args) {
        smsnotification obj1 = new smsnotification();
        emailnotification obj2 = new emailnotification();
        Notification notification;
        notification = obj1;
        System.out.println("sms sent"notification.notifyuser());

             }

    }







package rocky;

public interface Notification {
    void notifyuser();
}

class smsnotification implements Notification{

    @Override
    public void notifyuser() {
        System.out.println("we are sending text messages");
    }
}
class emailnotification implements Notification{

    @Override
    public void notifyuser() {
        System.out.println("we are sending email");
    }
}

**Reply by rockyrkl99** [reply]
my name is rakesh
